packages/components/psammead-navigation/src/Dropdown/index.jsx

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string, node, bool, func, oneOf } from 'prop-types';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { navigationIcons } from '@bbc/psammead-assets/svgs';
+import {
+  C_WHITE,
+  C_POSTBOX,
+  C_EBON,
+  C_SHADOW,
+} from '@bbc/psammead-styles/colours';
+import {
+  GEL_SPACING_HLF,
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const MENU_ICON_SIDE_LENGTH = '2.75rem'; // 44px
+
+const getStyles = dir => {
+  const direction = dir === 'ltr' ? 'left' : 'right';
+  return `border-${direction}: ${GEL_SPACING_HLF} solid ${C_WHITE};
+          padding-${direction}: ${GEL_SPACING}`;
+};
+
+const DropdownWrapper = styled.div`
+  background-color: ${C_EBON};
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    display: none;
+  }
+`;
+
+const CrossButton = styled.button`
+  background-color: ${C_EBON};
+  width: ${MENU_ICON_SIDE_LENGTH};
+  height: ${MENU_ICON_SIDE_LENGTH};
+  padding: 0;
+  margin: 0;
+  border: 0;
+
+  &:focus {
+    box-shadow: inset 0 0 0 ${GEL_SPACING_HLF} ${C_WHITE};
```

Comment:
This doesn't look to work fine with high contrast in FF. 

![focus-border-hamburger](https://user-images.githubusercontent.com/46446236/69325545-3ed8ec80-0c42-11ea-86c5-a28ed88c8a2c.png)
![focus-border-cross](https://user-images.githubusercontent.com/46446236/69325553-3f718300-0c42-11ea-9338-05924783f847.png)

You could use a pseudo-element as we do in the Navigation:
```
  &:focus::after {
    border: ${GEL_SPACING_HLF} solid ${C_WHITE};
  }
```